skietbaan_fe/src/components/Login.js

```diff
@@ -1,98 +1,135 @@
 import React, { Component } from 'react';
 import {
-  Container, Col, Form,
-  FormGroup, Label, Input,
-  Button,
+  Container, Col,
+  FormGroup, Label, Input,Button,Form
 } from 'reactstrap';
 import '../components/LoginStyles.css';
 
+  function validatePassword(str) {
+      var re = /(?=.*\d)(?=.*[a-z]).{6,}/;
+      return re.test(str);
+    }
+
+    function validateEmail(email) {
```

Comment:
Having a closer look at the regex function, it passes this data(laurent@laurent.laurent) and considers it an email while it is not a valid email.

Reply:
alright, thanks